client/test/unit/node/index.ts

```diff
@@ -0,0 +1,52 @@
+import glob from "glob";
+import Mocha from "mocha";
+import path from "path";
+import process from "process";
+
+function run(): Promise<void> {
+  const mocha = new Mocha({
+    ui: "bdd",
+    color: true,
+    timeout: 100000,
+  });
+
+  console.log("scnjdl args", JSON.stringify(process.argv));
```

Comment:
please remove personal debug code

Reply:
> please remove personal debug code

Not related specifically to this PR, but would love for us to eventually stop using console.log in favor of console.info, console.warn, console.error etc.